index.js

```diff
@@ -22,60 +22,137 @@ const copyWithTemplate = async (from, to, variables) => {
 	await writeFile(to, generatedSource);
 };
 
-const fromPath = file => path.join(__dirname, 'template', file);
-const toPath = file => path.join(process.cwd(), file);
+const getTemplateFlag = typescript => {
+	if (!typescript) {
+		return 'javascript';
+	}
+
+	return 'typescript';
+};
+
+const install = async ({typescript}) => {
+	const unpack = dependencies => {
```

Comment:
To avoid this trick you can define `jsDependencies` and `jsDevDependencies` arrays instead (same for TypeScript ones). Then later:

```js
if (typescript) {
  await execa('npm', ['install', ...typescriptDependencies]);
  await execa('npm', ['install', '--save-dev', ...typescriptDevDependencies]);
} else {
  // Same for JS
}
```